docs/src/content/pages/collections-and-singletons.mdoc

```diff
@@ -6,7 +6,7 @@ Keystatic has two _concepts_ or mechanisms to manage data: `collections` and `si
 
 ## Collections
 
-Think of a `collection` of anything you’d want multiple instances of.
+Think of a `collection` of anything you'd want multiple instances of.
```

Comment:
Found another small (existing) one!
```suggestion
Think of a `collection` as anything you'd want multiple instances of.
```

Reply:
On line 13 there is also a typo: 'Kesytatic'. Since it isn't part of the diff Github won't let me make a suggestion on it